bobtimus/src/loan.rs

```diff
@@ -51,8 +52,8 @@ pub struct LoanOffer {
     /// Interest in relation to terms
     pub terms: Vec<Term>,
 
-    /// Interest rates in relation to collteralization
-    pub collateralizations: Vec<Collateralization>,
+    /// Interest rates in relation to ltv
```

Comment:
The `Ltv` struct does not represent a relation to the interest rate. I think this comment is incorrect.